### PR TITLE
Add customizable variable auth-source-pass-port-separator

### LINKED ISSUE #86: Feature Request: Support alternate separator character between host and port
I'm using password-store (actually [gopass](https://www.gopass.pw/)) on Windows as well as on Unix.

Windows does not allow filename to contain colon (`:`), so it would be nice if `auth-source-pass` can use alternate character (e.g. `#`?) as separator between host and port.

### PR DESCRIPTION
**Pull request by iquiw**
Fixes #86

### CODE REVIEW

File: auth-source-pass.el [hunk 1]
```diff
@@ -47,6 +47,10 @@
   "Path to the password-store folder."
   :type 'directory)
 
+(defcustom auth-source-pass-port-separator ":"
+  "Separator string between host and port in password filename."
```
Comment (DamienCassou) [inline auth-source-pass.el]: I think "entry" is more appropriate than `password` here.

**Comment by DamienCassou** [comment]
Have you signed the FSF copyright assignment form? Because auth-password-store is included in Emacs core, this is required.

File: auth-source-pass.el (hunk 1 above)
Comment (iquiw) [reply auth-source-pass.el]: Thanks for review!
I updated the docstring.

**Comment by iquiw** [comment]
> Have you signed the FSF copyright assignment form?

Yes. I have done my paper work when contributing to https://github.com/abo-abo/swiper/pull/1745#issuecomment-442693376

**Comment by DamienCassou** [comment]
Awesome, thank you very much.

**Pull request merged**